magento/magento2
# Fixed incorrect price of customizable options shown below "Customer Activities" #26101

### Description (*)
Use the `getFinalPrice()` on the product model instead of using the price info model to get the final price, it didn't seem the include the price of each customized option in the final price.

Refactor the implementation of the block for the "Shopping Cart" list below "Customer Activities" so that it uses the parents implementation if no custom price is found, this way I didn't have to duplicate the changes from the parent block.

### Fixed Issues (if relevant)
1. fixes magento/magento2#26101: Customizable Options Price

### Manual testing scenarios (*)
1. Create a product with at least one customizable option
2. Create a customer
3. Sales > Orders > Create New Order
4. Choose your customer
6. Add your product with at least one customizable option
7. "Items Ordered > Action > Move to Wishlist > Update Items and Quantities"
8. "Price" in the left sidebar underneath "Wish List" should include the price of the customizable option

### Contribution checklist (*)
